Assignment-1-6/form.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+include __DIR__ . '/emailValidate.php';
+
+$firstName = $lastName = $marksStr = '';
+
+$targetFile = '';
+$imgUpload = '';
+
+$subMarkArr = '';
+
+$phone = '+91';
+$validPhone = TRUE;
+
+$email = '';
+$emailCheck = '';
+
+
+// If Form submitted
+if (isset($_POST['submit'])) {
+
+  // Get firstName & lastName
+  $firstName = $_POST['first-name'];
+  $lastName = $_POST['last-name'];
```

Comment:
All variables should be snake cased inside non-class php files.
Please fix this across all files.